docs/product-assembly_modeling-processes/src/main/asciidoc/pizza-confirm-task-proc.adoc

```diff
@@ -3,26 +3,30 @@
 Create an order confirmation user task, which will return the order details to the customer for confirmation before the final order is placed.
 
 .Procedure
-. Create an outgoing connection from the Process Order user task to user task.
+. Click on the Process Order task to create an outgoing connection to a new user task.
 +
-image::create-task3.png[]
+.Outgoing connection from the Process Order task to a user task
+image::create-task3.png[Creating an outgoing connection from the Process Order task to a user task]
 
 +
-image::user_task3.png[]
+.Convert into a User task
+image::user_task3.png[Converting in to a user task]
 
-. In the upper-right corner, click the *Diagram properties* image:diagram_properties.png[] icon.
+. In the upper-right corner of the the diagram editor, click the *Diagram properties* image:diagram_properties.png[] icon.
```

Comment:
Search all "editor" and replace with "designer". Make sure in each case. That's the correct term since 7.0. Consider also being more specifying if it's a specific designer, like BPMN designer or whatever the panel is called in the left nav menu when you're looking at all your assets ("guided decision tables" panel, "decision tables" panel, "BPMN" panel, etc., are all the names used with "designer", like "In the guided rules designer, ..."). Not sure how much this applies or to what extent you care.

At very least, just change editor to designer I'd say, for consistency.

Reply:
Done